Demo/DemoChat/Sources/UI/TextToSpeechView.swift

```diff
@@ -0,0 +1,156 @@
+//
+//  File.swift
+//
+//
+//  Created by Ihor Makhnyk on 16.11.2023.
+//
+
+import SwiftUI
+import OpenAI
+import UIKit
+
+public struct TextToSpeechView: View {
+    
+    @ObservedObject var store: MiscStore
+    
+    @State private var prompt: String = ""
+    @State private var voice: AudioSpeechQuery.AudioSpeechVoice = .alloy
+    @State private var speed: Double = 1
+    @State private var responseFormat: AudioSpeechQuery.AudioSpeechResponseFormat = .mp3
+    
+    private let formats: [AudioSpeechQuery.AudioSpeechResponseFormat] = [.mp3, .aac, .flac, .opus]
```

Comment:
Because all of the properties here a related to the current view and current "ViewModel" is `MiscStore` which handles all the logic of "Misc" tab. Though I agree that it would be better to follow MVVM in a proper way, so it will come with a major update of the demo app, which has to be refactored and improved in a lot of ways.